toqito/state_props/tests/test_is_separable.py

```diff
@@ -208,3 +219,940 @@ def test_separable_based_on_eigenvalues():
         ]
     )
     np.testing.assert_equal(is_separable(rho), True)
+
+
```

Comment:
simialr to my comment in the other PR, if multiple unit tests follow the same pattern, it might be better to condense them via parameterize.